docs/low-code-spark/gems/source-target/warehouse/snowflake.md

```diff
@@ -31,10 +31,11 @@ Please refer the snowflake official documentation to attach private key to user
 | Url                    | Hostname for your account in the format: `<account_identifier>.snowflakecomputing.com`. <br/> Eg: `https://DJ07623.ap-south-1.aws.snowflakecomputing.com` | True                                                   |
 | Database               | Database to use for the session after connecting                                                                                                          | True                                                   |
 | Schema                 | Schema to use for the session after connecting                                                                                                            | True                                                   |
-| Warehouse              | The default virtual warehouse to use for the session after connecting                                                                                     | False                                                  |
+| Warehouse              | Default virtual warehouse to use for the session after connecting                                                                                         | False                                                  |
+| Role                   | Default security role to use for the session after connecting                                                                                             | False                                                  |
 | Data Source            | Strategy to read data: `DB Table` or `SQL Query`.                                                                                                         | True                                                   |
-| Table                  | The name of the table to be read. All columns and records are retrieved (i.e. it is equivalent to `SELECT * FROM table`).                                 | Required if `Data Source` is `DB Table`                |
-| SQL Query              | The exact query (`SELECT` statement) to run                                                                                                               | Required if `Data Source` is `SQL Query`               |
+| Table                  | Name of the table to be read. All columns and records are retrieved (i.e. it is equivalent to `SELECT * FROM table`).                                     | Required if `Data Source` is `DB Table`                |
+| SQL Query              | Exact query (`SELECT` statement) to run                                                                                                                   | Required if `Data Source` is `SQL Query`               |
```

Comment:
Can you add if the statement can ONLY be a Select Statement or can one run anything?

Reply:
it can be only select statement only